activerecord/test/models/company.rb

```diff
@@ -26,6 +26,9 @@ def arbitrary_method
   def private_method
     "I am Jack's innermost fears and aspirations"
   end
+
+  class SpecialCo < Company
+  end
```

Comment:
Ah, indeed [there is](https://github.com/rails/rails/blob/db0e649aba7135133154a35549219cffe968eaf6/activerecord/test/models/company.rb#L43)! That causes `find_sti_class` to blow up.